### ISSUE TITLE
Problem with fonts in public gateway

### ISSUE THREAD

**Issue by magaiverpr** [pglombardo/PasswordPusher#2574]
## 🔬 How To Reproduce

I have 2 enviroments to Password Push: DEV and PROD.

In DEV, I am not using the Public Gateway, and its all fine.

In PROD, where I use the Public Gateway, there is problems with the fonts and icons, like the print below:

### Environment

Where are you running/using Password Pusher?

- [ ] pwpush.com
- [ ] Docker Image
  - [x] pwpush
  - [ ] Custom image
- [ ] Heroku
- [ ] Digital Ocean
  - [ ] App Platform
  - [ ] Kubernetes Service
- [ ] Microsoft Azure
  - [ ] App Service
  - [ ] Container Instances (ACI)
  - [ ] Kubernetes Service (AKS)
- [ ] Google Cloud
  - [ ] App Engine
  - [ ] Cloud Run
  - [ ] Kubernetes Engine
- [ ] AWS
  - [ ] Elastic Container Service (ECS)
  - [ ] Kubernetes Service (AKS)
- [ ] Source Code
- [ ] Other (please specify)

If applicable, what version of Password Pusher?

Using kubernetes on-premise, and the Public Gateway is a docker in swarm. Both are using latest version.

### Screenshots

![image](https://github.com/user-attachments/assets/64a29d25-f74a-424c-9913-3fcbbacaf418)

![image](https://github.com/user-attachments/assets/7fb6d1d4-29d0-42e3-9255-df0677fd0dbb)

**Comment by pglombardo** [comment]
Hrm that's odd.  The fonts should be bundled in.  I'll try to dig into this over the weekend.

**Comment by pglombardo** [comment]
Hi @ozovalihasan - could you take a look at this issue and see if this still exists in the public gateway container image?  Notice the button icons are missing too.

**Comment by ozovalihasan** [comment]
Hi @pglombardo,

I have tested it by using a docker compose file on Chrome, Firefox and Opera. I haven't noticed any issue/difference between `pglombardo/pwpush` and `pglombardo/pwpush-public-gateway`.


```
# docker-compose.yml
services:
  db:
    image: docker.io/postgres:16
    volumes:
      - ./postgres-data:/var/lib/postgresql/data
    ports:
      - "5432:5432"
    restart: unless-stopped
    environment:
      POSTGRES_USER: pwpush_user
      POSTGRES_PASSWORD: pwpush_passwd
      POSTGRES_DB: pwpush_db

  pwpush:
    image: docker.io/pglombardo/pwpush:latest
    environment:
      DATABASE_URL: 'postgres://pwpush_user:pwpush_passwd@db:5432/pwpush_db'
    ports:
      - "5100:5100"
    restart: unless-stopped
    depends_on:
      - db
    links:
     - db:db

  pwpush-gateway:
    image: docker.io/pglombardo/pwpush-public-gateway:latest
    environment:
      DATABASE_URL: 'postgres://pwpush_user:pwpush_passwd@db:5432/pwpush_db'
    ports:
      - "5101:5100"
    restart: unless-stopped
    depends_on:
      - db
    links:
     - db:db
```

<img width="796" height="533" alt="Image" src="https://github.com/user-attachments/assets/19d0d85f-78cf-40bd-8c7a-76fd47eb002f" />

Please let me know if I'm missing any point.

**Comment by pglombardo** [comment]
Thanks @ozovalihasan - I think this was a transient issue that has since been resolved.  @magaiverpr I'll close this issue out soon unless you say otherwise.  Please let me know if you are still having this issue.

**Comment by magaiverpr** [comment]
Yes, we can close this issue. Sorry, I forget about this issue, I checked now and was alright.

Thanks!
